portfolio/src/main/webapp/Controller.js

```diff
@@ -0,0 +1,77 @@
+import CommentHandler from '/CommentHandler.js';
+import UserHandler from '/UserHandler.js';
+
+export default class Controller {
+
+  constructor(document) {
+    this.username = undefined;
+    this.loggedIn = false;
+    this.document = document;
+  }
+
+  /** Shows the change name form*/
+  openNameForm() {
+    this.document.getElementById('name-form').classList.remove('hidden-elem');
+  }
+
+  /** Hides the change name form*/
+  closeNameForm() {
+    this.document.getElementById('name-form').classList.add('hidden-elem');
+  }
+
+  /**
+   * Loads all comments from backend.
+   * This function is called when the page is loaded.
+   * @param {HTML div object} commentBox The div in document with id "comment-box"
+   * @param {HTML template} commentCell The template for one comment
+   */
+  static async loadComments(commentBox, commentCell) {
```

Comment:
This shouldn't be a static method. You can make it a private method (`loadComments_`) and only call it from within this Controller class. Static methods are pretty rare in the normal course of things.